### PR TITLE
Use a json config as argument for controller

### PR DESCRIPTION
**Pull request by stano45**
- Pass `switches.json` as argument to controller to configure switches
- Add `controller` Makefile target to each example
- Remove `control` Makefile target in `/utils/Makefile`
- Move runtime files from load balancer to examples
- This PR exists to make it easier to change the controller config in future examples (e.g. redis container migration)

@rst0git

### CODE REVIEW

File: examples/host_containers/Makefile [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
-all: compile build control_multi_switch
+all: compile build controller
 
 terminals: h1 h2 h3 h4
 
 build: clean
 	./build.sh
 
+controller:
+	sleep 2 && ../../controller/controller.py --config config/switches.json
```
Comment (rst0git) [inline examples/host_containers/Makefile:11]: Would it be possible to add a comment describing why we use `sleep` here?

File: controller/controller.py [hunk 2]
```diff
@@ -121,42 +115,16 @@ def update_node():
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description="P4Runtime Controller")
     parser.add_argument(
-        "--p4info",
-        help="p4info proto in text format from p4c",
+        "--config",
+        help="JSON configuration file for switches",
```
Comment (rst0git) [inline controller/controller.py:119]: Would it be possible to add more information in the commit message that describes why `--p4info` is removed?

File: examples/host_containers/Makefile (hunk 1 above)
Comment (stano45) [reply examples/host_containers/Makefile:12]: Sometimes the switches have not been initialized yet when the controller attempts to program them. A 2-second wait seems to be enough to wait for the switch initialization. I added a comment to the target.

File: controller/controller.py (hunk 2 above)
Comment (stano45) [reply controller/controller.py:119]: I added a longer description to the commit message. Essentially, the switch configuration was hard-coded in the controller. This change allows for the configuration of a flexible number of switches the configuration of the runtime file and further options for each switch. This is used in a later PR with the Redis example - https://github.com/stano45/p4containerflow/pull/18/files#diff-3a68632aa395a2d8d53226b541c60893fab8e0f66b8f1208fa564e25e57cd45d.

**Comment by rst0git** [comment]
LGTM

**Pull request merged**